src/_utils/_git_secret_tools.sh

```diff
@@ -605,6 +605,7 @@ function _get_user_key_expiry {
 function _assert_keychain_contains_emails {
```

Comment:
Can we implement this by creating a function called _asset_keychain_contains_email_once(), instead of using this 'force' flag here?